### ISSUE TITLE
`failOnAccessibilityError` does not work with `baselinefile`

### ISSUE THREAD

**Issue by astherath** [microsoft/accessibility-insights-action#1815]
## Describe the bug
The `failOnAccessibilityError` flag does not work with `baselinefile` flag. If the `failOn` flag is set to `false` it will still fail the pr step if there is also a baseline file.

**To Reproduce**

1. Set both `failOnAccessibilityError` to `false` and `baselinefile` flag to be non-empty
2. The pr step will fail if there are new a11y bugs.

## Expected behavior
If the `failOnAccessibilityError` flag is set to `false` then even if the baseline file is changed from the scan output, it would not fail the PR.

- Azure DevOps Extension Version & Environment: 2.0.6

ADO PR link: https://msazure.visualstudio.com/One/_build/results?buildId=79703612&view=logs&j=ee348bfc-8e0c-5bd0-a505-e5b7da3254f4&t=01208c52-d4f4-53a5-cc39-7e3b2f04a543

## Are you willing to submit a PR?

Possibly if small

## Did you search for similar existing issues?

Yes

**Comment by DaveTryon** [comment]
Hi, @astherath! The `failOnAccessibilityError` and `baselineFile` options control two _independent_ failure modes that are really intended for different things. If the `baselineFile` option is specified, then the task will report a failure any time the currently generated differs from the previously generated baseline. If the `failOnAccessibilityError` option is `true`, then the task will report a failure any time an accessibility error is detected in the current results. Settings `failOnAccessibilityError` to `false` intentionally has no impact on any failures reported as a result of the `baselineFile` option. I'll update the docs to clarify this difference.

Thanks for using Accessibility Insights!
